tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -95,3 +98,48 @@ After(function (testCase) {
   console.log('\n  Result: ' + testCase.result.status + '\n')
   createdFiles.forEach(fileName => fs.unlinkSync(fileName))
 })
+
+Before(async function () {
+  const resp = await occHelper.runOcc(
+    [
+      'config:list'
+    ])
+  let stdOut = _.get(resp, 'ocs.data.stdOut')
+  if (stdOut === undefined) {
+    throw new Error('stdOut notFound, Found:', resp)
+  }
+  stdOut = JSON.parse(stdOut)
+  initialAppConfigSettings = _.get(stdOut, 'apps')
+  if (initialAppConfigSettings === undefined) {
+    throw new Error("'apps' was not found inside stdOut of response.")
+  }
+})
+
+After(async function () {
```

Comment:
I think it would be better to do a diff of the objects which would simplify code here (but, maybe, I'm overthinking?).

What do you think?

Reply:
yes you are overthinking :dagger: